strawberryfields/backends/states.py

```diff
@@ -621,6 +622,23 @@ def wigner(self, mode, xvec, pvec):
 
         return W / (self._hbar)
 
+    def p_quad_values(self, mode, xvec, pvec):
+
+        W = BaseFockState.wigner(mode, xvec, pvec)
+        y = []
+        for i in range(0, len(xvec)):
+            res = simps([W[i][k] for k in range(0, len(xvec))], xvec)
+            y.append(res)
+        return np.array(y)
+
+    def x_quad_values(self, mode, xvec, pvec):
```

Comment:
If you add a docstring here (see the other methods for the style used), it will automatically show up in the docs!